ios/Scripts/update_icons.sh

```
@@ -0,0 +1,79 @@
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+PROJECT_DIR="$SCRIPT_DIR"
+ICONS_DIR="$PROJECT_DIR/PulsePlate/Assets.xcassets/AppIcon.appiconset"
+
+# Проверяем, что папка существует
+if [ ! -d "$ICONS_DIR" ]; then
+    echo "❌ Папка с иконками не найдена: $ICONS_DIR"
+    exit 1
+fi
+
+# Переходим в папку проекта
+cd "$PROJECT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)